installer/docker/Dockerfile

```diff
@@ -83,17 +90,18 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.schema-version=$VCS_TAG
 
-# install NEURON
-RUN sudo pip install NEURON
+# install hnn-core
+RUN pip install hnn-core
+
```

Comment:
We shouldn't need hnn-core on this branch.

Reply:
Gotcha, thanks!